generator/parseXML/ParseXML.py

```diff
@@ -184,24 +185,38 @@ def get_enum_name(self, node):
 
     @staticmethod
     def get_mapping(self, node):
+        '''
+        Used to get information from a <mapping> node
+        e.g.
+        <mapping name="DimensionDescription" package="numl"/>
+
+        :param node: the <mapping> node to interrogate
+        :return: returns [name, package] tuple
+        '''
         name = ''
-        package = ''
+        package = ''  # TODO Maybe better as None?
+
+        # Example values for `name`:
+        # "Transition", "QualitativeSpecies"
         temp = self.get_value(node, 'name')
-        # expect camelcase with upper first
+        # expect CamelCase with upper first
         if temp is not None:
             name = strFunctions.upper_first(temp)
+
+        # Example for `package`: "qual"
         temp = self.get_value(node, 'package')
-        # expect lower first
+        # Expect lower-case first letter:
         if temp is not None:
             package = strFunctions.lower_first(temp)
+
         return [name, package]
 
 
     @staticmethod
     def get_loclass_name_value(self, node, name):
         xml_loclass_name = ''
         temp = self.get_value(node, name)
-        # we expect this to be camel case starting with lower
+        # we expect this to be camelCase starting with lower
         if temp is not None:
             xml_loclass_name = strFunctions.upper_first(temp)
```

Comment:
Can you add an issue for this class as it says camelCase with lower first but code does upper_first

Reply:
Now issue 16